### DISCUSSION TITLE
Possible offline installation needed for cs_tools 1.5.3

### DISCUSSION THREAD

**Discussion by llucjan** [thoughtspot/cs_tools#125] [General]
Hi Team,

**What I'm trying to achieve?**

Install a new instance of cs_tools v1.5.3 

**What error I'm getting:**

After fire the installation command:
`curl --silent --show-error --location-trusted  https://raw.githubusercontent.com/thoughtspot/cs_tools/master/cs_tools/updater/_bootstrapper.py  | python3.11 - --install`

The result is : `Missing '_updater.py', downloading from GitHub`

![image](https://github.com/thoughtspot/cs_tools/assets/166501651/e7e0bd8c-6425-4dfd-9fd8-ea36789aa757)

**Similar issue**: #111 

**Manual approach to get the file from github**:
Response status code: 403
Response text: {"message":"API rate limit exceeded for x.y.z.182. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)","documentation_url":"https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"}

**System specific**
```
OS

NAME="Amazon Linux"
VERSION="2023"
ID="amzn"
ID_LIKE="fedora"
VERSION_ID="2023"
PLATFORM_ID="platform:al2023"
PRETTY_NAME="Amazon Linux 2023.4.20240401"

Python

base = "/usr"
datarootdir = "/usr/share"
exec_prefix = "/usr"
installed_base = "/usr"
installed_platbase = "/usr"
platbase = "/usr"
platlibdir = "lib64"
prefix = "/usr"
projectbase = "/usr/bin"
py_version = "3.11.6"
py_version_nodot = "311"
py_version_nodot_plat = ""
py_version_short = "3.11"
srcdir = "/usr/lib64/python3.11/config-3.11-x86_64-linux-gnu"
userbase = "/home/ec2-user/.local"
```

**Comment by boonhapus** [comment]
We got on a call together and worked this one out! The bootstrapper API call was failing silently and causing some issues, I've added some logging for this in the next version so it's more visible.
